modules/persistent-storage-csi-smb-cifs-create-sc.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-sc_{context}"]
+= Creating a storage class for SMB CIFS
+
+After installing the Operator, you should create a storage class for dynamic provisioning of Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect volumes.
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
```

Comment:
Let's two more prerequisites:

> * You have SMB server installed and know its:
>  - hostname
>  - share name
>  - username/password
> * You created a Secret called `smbcreds` in the namespace `samba-server` and put username and password as `stringData` there.

Reply:
I added the 1st prerequiste. I have the 2nd one as part of the procedure.